src/commands/version.ts

```diff
@@ -135,48 +99,28 @@ const shouldUpgradeVersion = async ({
 
   if (!latestVersion) {
     console.log(`${red(`Cannot extract version from asset "${assetKey}". Reach out Juno❗️`)}`);
-    return {upgrade: false};
+    return;
   }
 
   const diff = compare(currentVersion, latestVersion);
 
   if (diff === 0) {
-    return {upgrade: false};
+    console.log(`Your ${displayHint} (${green(`v${currentVersion}`)}) is up-to-date.`);
+    return;
   }
 
   if (diff === 1) {
     console.log(
       `${yellow(`'Your ${displayHint} version is more recent than the latest available 🤔.'`)}`
     );
-    return {upgrade: false};
+    return;
   }
 
-  const answer = await confirm(
-    `A new version ${latestVersion} for your ${cyan(
-      displayHint
-    )} is available.${NEW_CMD_LINE}Do you want to ${cyan('upgrade')} now?`
+  console.log(
+    `Your ${displayHint} (${yellow(`v${currentVersion}`)}) is behind the latest version (${green(
+      `v${latestVersion}`
+    )}) available. Run ${cyan(
+      `juno upgrade${assetKey === 'mission_control' ? '-m' : ''}`
```

Comment:
@peterpeterparker there is a space missing before `-m` so currently with version 0.21 I got something like this:

```shell
Your mission control (v0.0.2) is behind the latest version (v0.0.3) available. Run juno upgrade-m to update it.
Your satellite "web3marketing.ufostart.com" (v0.0.7) is up-to-date.
```

Watch out for **`upgrade-m `**

Reply:
Thanks! Fixed in https://github.com/buildwithjuno/cli/pull/16.

I'll wait few days to see if anything else comes in and then will release a new version of the cli.